src/interfaces/swig/Machine.i

```diff
@@ -7,6 +7,9 @@
 SHARED_RANDOM_INTERFACE(shogun::Machine)
 %shared_ptr(shogun::Machine)
 %shared_ptr(shogun::LinearMachine)
+%shared_ptr(shogun::NonParametricMachine)
 %shared_ptr(shogun::DistanceMachine)
 %shared_ptr(shogun::IterativeMachine<LinearMachine>)
 
+%include <shogun/machine/NonParametricMachine.h>
```

Comment:
So from the output of azure pipelines it seems like this has to come after ` %include <shogun/machine/Machine.h>`, so you'll have to find all the places with this line, and add `%include <shogun/machine/NonParametricMachine.h>` after that